src/pages/feedbacks/FeedbackList.vue

```diff
@@ -31,10 +38,20 @@ import FeedbackItem from "../../components/feedbacks/FeedbackItem.vue";
 import NoFeedback from "../../components/feedbacks/NoFeedback.vue";
 import Wrapper from "../../components/basicComponents/Wrapper.vue";
 // set the v-if - if there is no feedback add NoFeedback component v-else if there are feedbacks set FeedbackItem component with v-for
+// <!-- attach NoFeedback and make like in the BaseButton props with Boolean and if true attach feedback item if false - v-else - attach NoFeedback component
+defineProps({
+  noFeedback: {
+    type: Boolean,
+    defaullt: false,
+  },
+});
 const optionDetails = [
   "Most Upvotes",
   "Least Upvotes",
   "Most Comments",
   "Least Comments",
 ];
```

Comment:
You will need here pair of value / name

Reply:
I added this as a suggestion - I will ask you on Monday about that cause I am not sure if I understand well what do you mean